### PR TITLE
Updated evdi to 1.4.1 to support post 4.10 kernels

### PR DESCRIPTION
**Pull request by alhalama**

### CODE REVIEW

**Comment by ssaavedra** [comment]
You'd need to also modify the RELEASE field in .travis.yml, as you can see here: https://github.com/alhalama/displaylink-rpm/compare/master...displaylink-rpm:patch-pr-26

That's why Travis is failing to build.

File: .travis.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@ language: generic
 sudo: required
 env:
   global:
-    - VERSION=1.3.43
+    - VERSION=1.4.1
     - DAEMON_VERSION=1.3.52
     - RELEASE=2
```
Comment (ssaavedra) [inline .travis.yml]: Please change RELEASE=1 here too.

**Comment by alhalama** [comment]
I have updated RELEASE in .travis.yml.

**Comment by ssaavedra** [comment]
LGTM

Sorry for the delay

**Pull request merged**